src/fsharp/FSharp.Compiler.Private/FSharp.Compiler.Private.fsproj

```diff
@@ -702,20 +683,8 @@
     <ProjectReference Include="$(MSBuildThisFileDirectory)..\FSharp.Core\FSharp.Core.fsproj" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'net472'">
-    <Reference Include="ISymWrapper, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" />
-    <PackageReference Include="Microsoft.Build" Version="$(MicrosoftBuildVersion)" />
-    <PackageReference Include="Microsoft.Build.Framework" Version="$(MicrosoftBuildVersion)" />
-    <PackageReference Include="Microsoft.Build.Tasks.Core" Version="$(MicrosoftBuildVersion)" />
-    <PackageReference Include="Microsoft.Build.Utilities.Core" Version="$(MicrosoftBuildVersion)" />
-    <PackageReference Include="System.IO.Compression" Version="$(SystemIoCompressionVersion)" />
-  </ItemGroup>
-
-  <ItemGroup Condition="$(TargetFramework.Startswith('netstandard'))">
```

Comment:
I don't think System.Runtime.Loader is in netstandard2.0.  Although perhaps the Apis are now in System.Runtime.

Reply:
It's NS 1.5

Reply:
cool